app/views/robot_verifications/submission.html.erb

```diff
@@ -49,7 +50,7 @@
        <%   next_field = "destination_bed_barcodes_#{bed_label.to_i +1 }" %>
       <% end %>
       <td><%= text_field_tag "destination_plate_barcodes[#{destination_label}]", params.dig(:destination_plate_barcodes, destination_label), :onkeypress => "return disable_cr_and_change_focus(event,'destination_plate_barcodes_#{destination_label}','#{next_field}')" %></td>
-      <td><%= PlateType.cherrypickable_default_type %></td>
+      <td><%= select_tag("destination_plate_types[#{destination_label}]", options_for_select(plate_types, PlateType.cherrypickable_default_type)) %></td>
```

Comment:
Add the dropdown itself, defaulting to what it was previously hardcoded to. (Which will cover the vast majority of picks)